src/main/java/com/sngular/kloadgen/property/editor/NameStrategyPropertyEditor.java

```diff
@@ -55,13 +55,21 @@ private void init() {
     nameStrategyComboBox.addActionListener(this);
   }
 
-  private void fillSerializer(final JComboBox<String> objectJComboBox) {
+    private void fillSerializer(final JComboBox<String> objectJComboBox) {
+    objectJComboBox.addItem("io.apicurio.registry.serde.avro.strategy.RecordIdStrategy");
+    objectJComboBox.addItem("io.apicurio.registry.serde.avro.strategy.TopicRecordIdStrategy");
+    objectJComboBox.addItem("io.apicurio.registry.serde.strategy.TopicIdStrategy");
+    objectJComboBox.addItem("io.apicurio.registry.serde.strategy.SimpleTopicIdStrategy");
+    nameStrategyComboBox = objectJComboBox;
+    /*
+    // todo: refactorizar esto para obtener las estrategias tanto de Confluent como de Apicurio
```

Comment:
Please remove this commented code and the todo comment if its necessary write it in English

Reply:
Done